test/models/user_test.rb

```diff
@@ -572,7 +570,7 @@
             create(:activity, type_id: Activity.type_ids[activity_type], user_id: @user.id, created_at: @created_at)
           end
 
-          it "nilであること" do
+          it 'nilであること' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.